recipes/tip/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "tip" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4ba6f6a9e652fa48a99534997aaaa0921a94af0ef2d65ce909b6825c06a2b259
+
+build:
+  skip: true  # [win and vc<14 or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make 4.*  # [unix]
+    - ninja 1.*
+    - cmake 3.*
+    - pkgconfig 1.*
+
+  host:
+    - libpcap 1.*
```

Comment:
[The logs show libpcap is unused.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=606090&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1928) Also, static linking is not allowed. Switch to dynamic linking or remove this unused dependency.

Reply:
libpcap has been removed and all linking is dynamic.